### PR TITLE
Allow additional submitters update with CAN_MANAGE_METADATA

### LINKED ISSUE #2438: Additional submitters should be editable with `CAN_MANAGE_METADATA`
see https://github.com/OpenSlides/openslides-client/issues/3689

### PR DESCRIPTION
**Pull request by bastianjoel**
resolves #2438

### CODE REVIEW

**Review by luisa-beerboom — CHANGES_REQUESTED** [review]
In general changes to actions and presenters should be accompanied by the underlying changes in the docs (under openslides-backend/docs)

File: openslides_backend/presenter/get_forwarding_committees.py [hunk 1]
```diff
@@ -38,11 +38,11 @@ def get_result(self) -> Any:
         if not has_perm(
             self.datastore,
             self.user_id,
-            Permissions.Motion.CAN_MANAGE,
+            Permissions.Motion.CAN_MANAGE_METADATA,
             self.data["meeting_id"],
         ):
             msg = "You are not allowed to perform presenter get_forwarding_committees"
-            msg += f" Missing permission: {Permissions.Motion.CAN_MANAGE}"
+            msg += f" Missing permission: {Permissions.Motion.CAN_MANAGE_METADATA}"
```
[the remark this answers is not in the document]
Comment (bastianjoel) [reply openslides_backend/presenter/get_forwarding_committees.py:45]: > User with the group permission 'can manage motion metadata' should be able to see and edit the committee field.

Last point in the client issue.

**Comment by bastianjoel** [comment]
There is no doc for that presenter.

**Review by luisa-beerboom — CHANGES_REQUESTED** [review]
Only change the wiki entry of the motion update then, I'll create an issue for the presenter

**Review by luisa-beerboom — APPROVED** [review]

**Pull request merged**